.bazelrc

```diff
@@ -271,7 +271,7 @@ build:rocm --define=using_rocm_hipcc=true
 build:rocm --define=tensorflow_mkldnn_contraction_kernel=0
 build:rocm --repo_env TF_NEED_ROCM=1
 # Disable unused-result on rocm builds.
-build:rocm --copt="-Wno-error=unused-result"
+## build:rocm --copt="-Wno-error=unused-result"
```

Comment:
Remove?

Reply:
Because currently upstream one still has few unit tests failed, which we will fix respectively soon, and ROCm 5.2 is not updated yet, maybe we can keep it till all unit tests are fixed and ROCm5.2 is updated?